dev_tools/conftest.py

```diff
@@ -28,6 +29,8 @@
 from dev_tools import shell_tools
 from dev_tools.env_tools import create_virtual_env
 
+LOGGER = logging.getLogger(__name__)
```

Comment:
Nit: the convention in other Cirq files that use logger has been to use `_LOGGER`.